modules/nw-ovn-kubernetes-session-affinity.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+:_content-type: CONCEPT
+[id="nw-ovn-kubernetes-session-affinity_{context}"]
+= Session affinity
+Session affinity is a feature that applies to Kubernetes `services` objects. You can use `session affinity` if you want to ensure that each time you connect to a ServiceVIP:Port the traffic is always load balanced to the same backend. For more information, including how to set session affinity based on a client's IP address, refer to link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity[Session affinity].
+
+[discrete]
+[id="nw-ovn-kubernetes-session-affinity-stickyness-timeout_{context}"]
+== Stickiness timeout for session affinity
+The OVN-Kubernetes network plugin for {product-title} calculates the stickiness timeout for a `session` from a client based on the last packet. For example, if you run a curl 10 times, the sticky session timer starts from the tenth packet not the first. As a result if the client is continuously contacting the service, then the session will never timeout. The timeout kicks in when the service has not received a packet for link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-stickiness-timeout[`timeoutSeconds`].
```

Comment:
* "session" here either needs to be not marked up, or if you're talking about the actual object itself, then it should be "`Session` object. My guess here is that a normal general reference of "session" not marked up is fine.
* Adding comma
* Consider avoiding future tense "will" by my suggested rewrite, if the meaning remains the same
* Avoid "kicks in". I'd recommend "starts" or "begins" or similar
* We usually try to avoid just saying a parameter name like this as text. I'd recommend adding more around it. Added a suggestion below, but you can reword however makes sense

```suggestion
The OVN-Kubernetes network plugin for {product-title} calculates the stickiness timeout for a session from a client based on the last packet. For example, if you run a curl 10 times, the sticky session timer starts from the tenth packet not the first. As a result, if the client is continuously contacting the service, then the session never times out. The timeout starts when the service has not received a packet for the amount of time set by the link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-stickiness-timeout[`timeoutSeconds`] parameter.
```